app/views/questionnaire.py

```diff
@@ -43,11 +43,11 @@
 
 post_submission_blueprint = Blueprint(name='post_submission',
                                       import_name=__name__,
-                                      url_prefix='/questionnaire')
+                                      url_prefix='/questionnaire/<eq_id>/<form_type>/<collection_id>/')
```

Comment:
We don't need the Collection Exercise ID for analytics so should be removed. And therefore doesn't need to be added to session_data.